thread/src/components/FooterTabs.js

```diff
@@ -17,15 +17,15 @@ const FooterTabs = ({ navigationState }) => {
     const { index, routes } = navigationState;
 
     return (
-        <Footer>
-            <FooterTab>
+        <Footer style={{height: 27}}>
```

Comment:
using fixed heights again? please test in the future on small screens.

<img width="441" alt="screen shot 2018-07-26 at 11 39 36 pm" src="https://user-images.githubusercontent.com/22198040/43292397-f0b94748-912c-11e8-8f2f-6038c44145a6.png">